class.jetpack.php

```diff
@@ -988,6 +998,24 @@ public static function is_single_user_site() {
 	}
 
 	/**
+	 * Sync back wp_version
+	 */
+	public static function get_wp_version() {
+		global $wp_version;
+		return $wp_version;
+	}
+	/**
+	 * Keeps wp_version in sync with .com when WordPress core updates 
+	 **/
+	public static function update_get_wp_version( $update, $meta_data ) {
+		if ( 'update' === $meta_data['action'] && 'core' === $meta_data['type'] ) {
+			/** This action is documented in wp-includes/option.php */
+			/* This triggers the sync for the jetpack version */
+			do_action( 'add_option_jetpack_wp_version', 'jetpack_wp_version', (string) (bool) Jetpack::get_wp_version() );
```

Comment:
I'm pretty sure you didn't mean to use both `(bool)` and `(string)` here, because otherwise we'll always get `1` as a result.